evap/student/templates/student_index.html

```diff
@@ -16,7 +16,56 @@
             </div>
         </div>
     {% endif %}
-
+    {% if global_rewards %}
```

Comment:
Should we move this into its own file?

Reply:
Both inlined and in an extra file are fine for me here

Reply:
Never mind, was moved out in [7946709](https://github.com/e-valuation/EvaP/pull/2234/commits/79467098d0aa5e7774eb4a612aeacca831bf66e1). Resolving.